src/test/java/org/fogbowcloud/saps/engine/core/archiver/storage/NfsPermanentStorageTest.java

```diff
@@ -134,6 +139,18 @@ public void testDeleteNonExistentTaskDir() throws PermanentStorageException {
         permanentStorage.delete(task);
     }
 
+    @Test @Ignore
+    public void testGenerateAccessLinksTaskDir() throws PermanentStorageException {
+        properties.setProperty(SapsPropertiesConstants.NFS_PERMANENT_STORAGE_PATH, MOCK_NFS_STORAGE_PATH);
+        PermanentStorage permanentStorage = new NfsPermanentStorage(properties);
+
+        // Set this var
+        String taskId = "task-id";
+
+        // Print it and result check
+        permanentStorage.generateAccessLinks(taskId);
```

Comment:
Add assert to output from generateLink method.

Reply:
I believe there is no problem in removing the `@Ignore` annotation. Remove it.

Reply:
Done